js/components/portal-dashboard/feedback/activity-level-feedback-student-rows.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Map } from "immutable";
+import { getFormattedStudentName } from "../../../util/student-utils";
+import AwaitingFeedbackActivityBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-activity-badge-icon.svg";
+import GivenFeedbackActivityBadgeIcon from "../../../../img/svg-icons/given-feedback-activity-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-rows.less";
+
+interface IProps {
+  activityId: string | null;
```

Comment:
Is this really sometimes null or is it meant to be optional?  If so we can write:
`activityId?: string`